src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/AdminTranscriptionSearchServiceImpl.java

```diff
@@ -52,6 +63,23 @@ public List<TranscriptionSearchResponse> searchTranscriptions(TranscriptionSearc
             .toList();
     }
 
+    @Override
+    public List<GetTranscriptionDetailResponse> getTranscriptionsForUser(Integer userId, OffsetDateTime requestedAtFrom) {
+        List<GetTranscriptionDetailResponse> detailResponseList = new ArrayList<>();
+
+        List<TranscriptionEntity> entityList = transcriptionRepository.findTranscriptionForUserOnOrAfterDate(userId, requestedAtFrom);
+
+        if (entityList.isEmpty()) {
+            throw new DartsApiException(TRANSCRIPTION_NOT_FOUND);
+        }
```

Comment:
why would this be an exception?
Can't we just return a result with an empty list?

Reply:
as per your pervious comment this has been changed